core/src/main/java/org/axonframework/commandhandling/gateway/CommandGatewayFactory.java

```diff
@@ -98,67 +98,38 @@
     private final CommandBus commandBus;
     private final RetryScheduler retryScheduler;
     private final List<MessageDispatchInterceptor<? super CommandMessage<?>>> dispatchInterceptors;
+
     private final List<CommandCallback<?, ?>> commandCallbacks;
 
     /**
-     * Initialize the factory sending Commands to the given {@code commandBus}, optionally intercepting them with
-     * given {@code dispatchInterceptors}.
-     * <p/>
-     * Note that the given {@code dispatchInterceptors} are applied only on commands sent through gateways that
-     * have been created using this factory.
+     * Instantiate a {@link CommandGatewayFactory} based on the fields contained in the {@link Builder}.
+     * <p>
+     * Will assert that the {@link CommandBus} are not {@code null}, and will throw an
```

Comment:
`are not {@code null}` -> `is not {@code null}`

Reply:
Thanks for that..